[CI] Importing translations failing

https://github.com/rm3l/maoni/actions/runs/9251242633/job/25446347923

```
Run ./tools/import-translations-github.sh
+ '[' '' == '' ']'
Starting translation import...
+ echo -e 'Starting translation import...\n'
+ cd /home/runner
+ git config --global user.email maoni+github_actions@rm3l.org

+ git config --global user.name 'Maoni Github Actions'
+ git clone --branch=master ***github.com/rm3l/maoni.git master
Cloning into 'master'...
+ cd master
+ wget -q https://downloads.crowdin.com/cli/v2/crowdin-cli.zip
+ unzip -qj crowdin-cli.zip
+ java -jar crowdin-cli.jar upload sources
ERROR
code: 2
message: Version [1](https://github.com/rm3l/maoni/actions/runs/9251242633/job/25446347923#step:5:1).0 of the Crowdin API has been deprecated and removed
Error: Process completed with exit code 1.
```

--- a/tools/import-translations-github.sh
+++ b/tools/import-translations-github.sh
@@ -36,7 +36,7 @@ if [ "$CIRCLE_PULL_REQUEST" == "" ]; then
   git clone --branch=master https://rm3l:$GITHUB_API_KEY@github.com/rm3l/maoni.git master > /dev/null
 
   cd master
-  wget -q https://downloads.crowdin.com/cli/v2/crowdin-cli.zip
+  wget -q https://github.com/crowdin/crowdin-cli/releases/3.19.4/download/crowdin-cli.zip
   unzip -qj crowdin-cli.zip
   # Update sources
   java -jar crowdin-cli.jar upload sources

